runner/README.md

```diff
@@ -0,0 +1,57 @@
+# Runner Design #
+package runner is structs and interfaces to run Scoot processes. package runners contains implementations. This doc describes both.
```

Comment:
we should establish key terms at the top of the readme and use them consistently throughout (eg: command, runRequest, etc.).

Package 'runner' is structs and interfaces to run a user defined (unix) command.  Package 'runners' contains an implementation.  This doc describes both